### PR TITLE
Added focus

### PR DESCRIPTION
**Pull request by jhkimnew**
https://github.com/Microsoft/IIS.WebManager/issues/349

Selecting feature should be done automatically when the feature got focused.

### CODE REVIEW

File: src/app/common/feature-vtabs.component.ts [hunk 1]
```diff
@@ -47,13 +48,10 @@ export class GlobalModuleReference {
         <item *ngFor="let module of features" [name]="module.name" [ico]="module.ico" [category]="subcategory">
             <dynamic [name]="module.component_name" [module]="module" [data]="module.data"></dynamic>
         </item>
-        <ng-container *ngFor="let module of contexts">
-            <item [name]="module.name" [ico]="module.ico" [category]="'${HomeCategory}'" [routerLink]="module.routerLink">
```
Comment (shhsu) [inline src/app/common/feature-vtabs.component.ts:51]: keep *ngIf="!(module.routerLink)"

File: src/app/common/vtabs.component.ts [hunk 2]
```diff
@@ -24,7 +24,8 @@ import { IsWAC } from 'environments/environment';
                         [ngClass]="{active: tab.active}"
                         (keyup.space)="selectItem(tab)"
                         (keyup.enter)="selectItem(tab)"
-                        (click)="selectItem(tab)">
+                        (focus)="selectItem(tab)"
```
Comment (shhsu) [inline src/app/common/vtabs.component.ts]: This could be a performance issue, need testing

File: src/app/common/vtabs.component.ts [hunk 3]
```diff
@@ -24,7 +24,8 @@ import { IsWAC } from 'environments/environment';
...
+                        (click)=" IsWAC? selectItem(tab): '' ">
```
Comment (shhsu) [inline src/app/common/vtabs.component.ts]: We should keep the behavior the same for WAC and site

Comment (jhkimnew) [reply src/app/common/vtabs.component.ts]: Done.

**Review by shhsu — APPROVED** [review]

**Pull request merged**